doc/website/getting-started/installation.md

```diff
@@ -79,6 +79,20 @@ X86_64:
 !!! attention
     Please ensure that the folder `/var/lock` exist and the filesystem supports file locking.
 
+### :fontawesome-brands-windows: Windows
+
+In case you do not have a Windows installation, Microsoft provides free developer images from [here](https://developer.microsoft.com/en-us/windows/downloads/virtual-machines/).
+
+Additionally, [CMake](https://cmake.org/download/) and [git](https://gitforwindows.org/) are required. The option to add CMake to the system PATH for all users should be set when it is installed.
+
+If the developer image from Microsoft is used, Visual Studio Community 2019 is already installed, else it can be found [here](https://visualstudio.microsoft.com/de/downloads/).
+
+To be able to compile iceoryx, the `Desktop development with C++` Workload must be installed. This is done by running `VisualStudioInstaller` and selecting the `Modify` button on `Visual Studio Community 2019`.
+
+Either `VS Code` or `Developer Command Prompt` can be used to build iceoryx with CMake. Maybe one or two restarts are required to let CMake find the compiler.
```

Comment:
If you want to use the Visual Studio IDE i would recommend to create a CMake Configuration for iceoryx_meta to build iceoryx for msvc in Debug/Release Mode.

Reply:
I would leave this as an exercise to the user since we also do not explain how to integrate it into other IDEs. With Linux as well as with macOS we only describe how to build it from the command line.